src/analysis/retail/monk/mistweaver/modules/spells/VivaciousVivify.tsx

```diff
@@ -21,20 +27,38 @@ class VivaciousVivification extends Analyzer {
       Events.refreshbuff.to(SELECTED_PLAYER).spell(SPELLS.VIVIFICATION_BUFF),
       this.onRefresh,
     );
+    this.addEventListener(
+      Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.RENEWING_MIST_HEAL),
+      this.onApplyRem,
+    );
+    this.addEventListener(
+      Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.RENEWING_MIST_HEAL),
+      this.onRemoveRem,
+    );
+  }
+
+  onApplyRem(event: ApplyBuffEvent) {
+    this.currentRenewingMists += 1;
+  }
+
+  onRemoveRem(event: RemoveBuffEvent) {
+    this.currentRenewingMists -= 1;
   }
 
   onRefresh(event: RefreshBuffEvent) {
     // every refresh is a wasted buff application and the CD restarts
-    this.wastedApplications += 1;
+    if (this.currentRenewingMists > this.vivify.estimatedAverageReMs) {
+      this.wastedApplications += 1;
+    }
   }
 
   get suggestionThresholds() {
     return {
       actual: this.wastedApplications,
       isGreaterThan: {
-        minor: 2,
-        average: 5,
-        major: 8,
+        minor: 0,
```

Comment:
I would be ok with a looser threshold here because of how fight dependent something like this is being on a static timer. It would be poor of us to count a wasted application solely because there are a lot of rems out if the raid is full health